_posts/2020-09-28-Kamen-Kanev-cv.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: cv
+title: Kamen Kanev
+comment: Default CV
+---
+# Kamen Kanev
+Software Engineer and Entrepreneur
+
+<div id="webaddress">
+<a target="_blank" rel="noopener" href="tel:+359879666995">+359-879-666-995</a>
+| <a target="_blank" rel="noopener" href="mailto:kamen.e.kanev@gmail.com">kamen.e.kanev@gmail.com</a>
+| <a target="_blank" rel="noopener" href="https://www.linkedin.com/in/kamen-kanev-97889b116/">LinkedIn</a>
+| <a target="_blank" rel="noopener" href="https://github.com/kanevk">Github</a>
+</div>
+
+## Emphasis
+
+Curious and open-minded individual, passionate about forming mental models for complex problems and
+developing solutions that have a social impact.
+Experienced in designing APIs, working on data-heavy applications, building processes, and managing people.
+
+### Technologies
+
+Ruby on Rails, React, GraphQL, PostgreSQL, Python, Django, NodeJS, RSpec, AWS, Capistrano
+
+## Professional Experience
+
+### Founder / CEO
+
+`April 2020 - September 2020`
+__[Bakalin BG](https://bakalin.bg/){:target="_blank" rel="noopener"}__
+
+We at Bakalin BG have the ambition of making the local specialty stores competitive to the supermarket chains.
+I and my partners have built the service from scratch, and I've learned a lot in various areas:
+Product vision, UX, Leadership, Logistics, Legal, Negotiation.
+One of the most noticeable achievements of mine is innovating with a delivery strategy
+that has improved our profit predictions with up to 20% on a yearly base.
+- Managing a team of 5 highly skilled young people
+- Contributing for 20-30% of the codebase: 2x React SPAs + Django GraphQL API
+- Developing a quantity management system(for grocery products) having up to 98% efficiency
+- Implementing agile practices in a **fully remote team** with a flexible work schedule
+- Making over 30 user interviews for validating product hypotheses
+
+### Team Lead
+
+`June 2019 - March 2020`
+__[Rifiniti, an FM:Systems Company](https://fmsystems.com/acq/rifiniti-is-now-fm-systems/){:target="_blank" rel="noopener"}__
+
+Working exclusively on Rails and NodeJS backed services. A Major part of my work included solving complex database problems
+coming from the requirements for fast analysis(<0.5s) of time-series data(over 500 million rows) for custom time ranges.
+Part of my contribution is:
+- Leading a team of 3 talented software engineers
+- Optimizing the load time of the main page with more than 30%
+- Speeding up the data deploys more than 2x
+
+### Senior Full-Stack Web Developer
+
+`May 2017 - May 2019`
+__[ReceiptBank](https://www.receipt-bank.com/){:target="_blank" rel="noopener"}__
+
+Worked alongside the best Rubists in town, maintaining a complex platform over Rails/GraphQL/Backbone/React tech stack.
```

Comment:
Don't use `in town`, as it is not used in formal documents. Maybe use 'experts in the sector'?

Reply:
`in the country`? Because I don't want to change the meaning.